src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -5055,17 +5023,17 @@ public void run() {
         GT_Values.RA.addMultiblockChemicalRecipe(
                 new ItemStack[] {
                     GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Bauxite, 16L),
-                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumCarbonate, 8L),
-                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Coal, 8L),
+                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumCarbonate, 16L),
+                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Coal, 16L),
                     GT_Utility.getIntegratedCircuit(24)
                 },
-                new FluidStack[] {Materials.Water.getFluid(1000L)},
+                new FluidStack[] {Materials.Water.getFluid(2000L)},
                 new FluidStack[] {
-                    Materials.Hydrogen.getGas(100L),
-                    Materials.CarbonDioxide.getGas(1750L),
-                    Materials.RedMud.getFluid(500L)
+                    Materials.Hydrogen.getGas(1000L),
+                    Materials.CarbonDioxide.getGas(2000L),
```

Comment:
The hydrogen seems fine.

It should be more equal towards the doubling, which would mean you should make it 3000L or 4000L.

Reply:
Done